### PR TITLE
Revamp of methodology page

### PR DESCRIPTION
**Pull request by vanshkumar**
After this PR, would like to make the "what data are missing" callout in the text into a link

### CODE REVIEW

**Comment by vanshkumar** [comment]
Decided to add in the metric definitions in this PR as well, working on that now

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx [hunk 1]
```diff
@@ -51,40 +51,111 @@ function MethodologyTab() {
...
+                <h4 className={styles.MethodologySubsubheaderText}>COVID-19</h4>
                 <p>
-                  Unfortunately, with these publically available data sets,
-                  there are crucial pieces missing, including but not limited
-                  to: comprehensive city-, census tract-, and county-level data;
-                  comprehensive race and ethnicity breakdowns; comprehensive
-                  gender and age breakdowns by county, etc.
+                  {/* TODO - this should really be a button... */}
+                  For a description of some of the gaps in COVID-19 data, please
+                  see the "What Data Are Missing" section on the "Explore the
```
Comment (adams314) [inline frontend/src/pages/DataCatalog/MethodologyTab.tsx:63]: Add link?

**Review by adams314 — APPROVED** [review]
LGTM but please check the adjustments I made

**Comment by vanshkumar** [comment]
> LGTM but please check the adjustments I made

All of your adjustments make sense, except for the very last one re: racial categories, the change was

Before:
Generally, all race/ethnicities on the tracker include Hispanic/Latino unless otherwise specified.

After:
Generally, all race/ethnicities on the tracker include Hispanic/Latino unless otherwise specified. A notable exception is Covid data, where cases, hospitalizations and deaths that are recorded as  Hispanic/Latino ignore any other race that is reported.

This applies to both Covid and BRFSS data, and also I don't think we need to call it out here as this is one of the examples presented in the list below

**Comment by adams314** [comment]
SGTM thanks!

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx (hunk 1 above)
Comment (vanshkumar) [reply frontend/src/pages/DataCatalog/MethodologyTab.tsx:63]: Planning to submit this first and then add it after (somewhat dependent on another PR)

**Pull request merged**